### PR TITLE
Ros2 v0.8.0 beta rm std msgs ssd traffic light

### PR DESCRIPTION
**Pull request by taikitanaka3**
rm float32 msgs and use autoware debug msgs instead

### CODE REVIEW

File: perception/traffic_light_recognition/traffic_light_ssd_fine_detector/include/traffic_light_ssd_fine_detector/nodelet.hpp [hunk 1]
```diff
@@ -36,8 +36,7 @@
 #include "cv_bridge/cv_bridge.h"
 #include "sensor_msgs/msg/image.hpp"
 #include "sensor_msgs/image_encodings.hpp"
-#include "std_msgs/msg/float32.hpp"
-#include "std_msgs/msg/header.hpp"
+#include "autoware_debug_msgs/msg/Float32Stamped.hpp"
```
Comment (tkimura4) [inline perception/traffic_light_recognition/traffic_light_ssd_fine_detector/include/traffic_light_ssd_fine_detector/nodelet.hpp:39]: #include "autoware_debug_msgs/msg/Float32Stamped.hpp"
->
#include "autoware_debug_msgs/msg/float32_stamped.hpp"

File: perception/traffic_light_recognition/traffic_light_ssd_fine_detector/src/nodelet.cpp [hunk 2]
```diff
@@ -79,7 +79,7 @@ TrafficLightSSDFineDetectorNodelet::TrafficLightSSDFineDetectorNodelet(const rcl
 
   std::lock_guard<std::mutex> lock(connect_mutex_);
   output_roi_pub_ = this->create_publisher<autoware_perception_msgs::msg::TrafficLightRoiArray>("output/rois", 1);
-  exe_time_pub_ = this->create_publisher<std_msgs::msg::Float32>("debug/exe_time_ms", 1);
+  exe_time_pub_ = this->create_publisher<autware_debug_msgs::msg::Float32Stamped>("debug/exe_time_ms", 1);
```
Comment (tkimura4) [inline perception/traffic_light_recognition/traffic_light_ssd_fine_detector/src/nodelet.cpp:82]: autware_debug_msgs
->
autoware_debug_msgs

Comment (taikitanaka3) [reply perception/traffic_light_recognition/traffic_light_ssd_fine_detector/src/nodelet.cpp]: fixed at ad4966b

File: perception/traffic_light_recognition/traffic_light_ssd_fine_detector/include/traffic_light_ssd_fine_detector/nodelet.hpp (hunk 1 above)
Comment (taikitanaka3) [reply perception/traffic_light_recognition/traffic_light_ssd_fine_detector/include/traffic_light_ssd_fine_detector/nodelet.hpp]: fixed at ad4966b

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**